crates/interpreter/src/module.rs

```diff
@@ -91,6 +76,14 @@ impl<'m> Module<'m> {
         module
     }
 
+    pub fn side_tables(&self) -> &[Vec<SideTableEntry>] {
+        self.side_tables
+    }
+
+    pub fn side_table(&self, i: usize) -> &[SideTableEntry] {
+        &self.side_tables[i]
+    }
```

Comment:
It seems safe to delete those functions now, since their type is wrong. We want to return `&'m [...]`.